Src/WitsmlExplorer.Api/Workers/BaseWorker.cs

```diff
@@ -59,6 +67,56 @@ protected string CancellationReason()
         {
             return "The job was cancelled by the user.";
         }
+        protected async Task<WitsmlCapServers> GetTargetServerCapabilities()
+        {
+            if (_targetServerCapabilities == null)
+            {
+                _targetServerCapabilities = await GetTargetWitsmlClientOrThrow().GetCap();
+            }
+            return _targetServerCapabilities;
+        }
+
+        protected List<WitsmlLogs> GetUpdateLogDataQueries(WitsmlLog log, WitsmlLogData logData, int chunkSize, string mnemonicList)
+        {
+            List<WitsmlLogs> batchedQueries = logData.Data.Chunk(chunkSize).Select(chunk =>
+                new WitsmlLogs
+                {
+                    Logs = new WitsmlLog
+                    {
+                        Uid = log.Uid,
+                        UidWell = log.UidWell,
+                        UidWellbore = log.UidWellbore,
+                        LogData = new WitsmlLogData
+                        {
+                            MnemonicList = mnemonicList,
+                            UnitList = logData.UnitList,
```

Comment:
You take the unitList from logData, but the mnemonicList as an input parameter. I think we should do the same for both for consistency.

Reply:
this is because in CopyLogDataWorker it is handled a little bit differently. In the solution you proposed, unit tests would fail.